docs/getting-started.md

```diff
@@ -58,29 +54,15 @@ Have a nice day! 👋
 
 ```
 
-You now will have the right `kubeconfig` for the kind cluster.
+### 2 - Install Weave GitOps onto the cluster
 
-Run `kubectl cluster-info --context kind-kind`
-
-### Install Weave GitOps onto the cluster
-
-2. Install Weave GitOps into the currently active Kubernetes cluster:
 ```console
 gitops install
```

Comment:
hmm, if we want to keep with using `add app .` and shared repo for application source and config, this will require us to shuffle the order and fork the repo at the start. On a separate note, it may be better to have the repo as a required parameter rather than a flag, seems much simpler for a user.

Reply:
@jrryjcksn based on the upcoming workshop, I'm going to address this in a future PR and push through changes into main and v0.4.1 docs